demos/smartlab_online_demo/python/README.md

```diff
@@ -0,0 +1,111 @@
+# Smartlab online Python\* Demo
+
+This is the demo applicatio with smartlab action recognition and smartlab object detection algorithms.
+The following pre-trained models are delived with the product:
+`
+
+* `smartlab-action-recognition-encoder-0001` + `smartlab-action-recognition-decoder-0001`, which are models for identifying actions of smartlab (2 actions). They recognize actions like adjust rider, put_take.
+
+* `smartlab-object-detection-0001` + `smartlab-object-detection-0002` + `smartlab-object-detection-0003` + `smartlab-object-detection-0004`, which are models for detecing smartlab objectsi (10 objects). They detect balance, weights, tweezers, box, battery, tray, ruler, rider, scale, hand.
+
+For more information about the pre-trained models, refer to the [Intel](../../../models/intel/index.md) and [public](../../../models/public/index.md) models documentation
+
+## How It works
+
+The demo pipeline consists of several steps, namely 'Decode', 'Detector', 'Segmentor', 'Evaluator', 'Display'.
+Every step is implemeted by correcponding class. Decode is implemented by opencv API, Detector is imlemeted `detector.py`. Segmentor is implemented by `segmentor.py`. Evaluator is implemented by `segmentor.py`. Evaluator is implemented by `evaluator.py`. Display is implemneted by `display.py`
```

Comment:
```suggestion
Every step is implemented in corresponding class.
```

Reply:
fixed

Reply:
I don't think you need this
> Every step is implemeted in correcponding class. Decode is implemented by opencv API, Detector is imlemeted `detector.py`. Segmentor is implemented by `segmentor.py`. Evaluator is implemented by `segmentor.py`. Evaluator is implemented by `evaluator.py`. Display is implemneted by `display.py`

You have good naming. And files names by themselves correspond to classes. So everything should be clear without explanation.

Reply:
removed